keyboards/willow/willow64/via/readme.md

```diff
@@ -0,0 +1,6 @@
+
+# Build with VIA enabled
+
+Make example for this keyboard (with VIA enabled):
+
+    qmk compile -kb willow/willow64/via -km default
```

Comment:
This file can be deleted.

Reply:
Unnecessary files and directories have been removed.